static/js/input.js

```diff
@@ -361,7 +362,8 @@ function parseFeaColumn(arr) {
  * @todo Terminate the search for categories and features if the entropy of
  * processed ones exceed a threshold, and return "description".
  */
-function guessColumnType(arr) {
+function guessColumnType(arr, threshold = 600) {
```

Comment:
Since you are assigning the default value in the next line, you don't need `= 600` in the function definition.